src/prpy/planning/ompl.py

```diff
@@ -245,3 +246,100 @@ def ShortcutPath(self, robot, path, timeout=1., **kwargs):
         }, append=True)
 
         return output_path
+
+class ConstrainedOMPLPlanner(OMPLPlanner):
+    def __init__(self):
+        OMPLPlanner.__init__(self, algorithm='CRRTConnect')
+
+    @ClonedPlanningMethod
+    def PlanToTSR(self, robot, tsrchains, planner_range=0.1, ompl_args=None, **kw_args):
+        """
+        Plan using the given TSR chains with OMPL. 
+        @param robot
+        @param tsrchains A list of TSRChain objects to respect during planning
+        @param planner_range the max stepsize for the planner before a projection
+         is applied
+        @param ompl_args ompl RRTConnect specific parameters
+        @return traj
+        """
+        if ompl_args is None:
+            ompl_args = {}
+        ompl_args['range'] = planner_range
```

Comment:
Why do we manually set `range` only for constrained planners?

Also, I think the default value should be derived from DOF resolution. We have some logic to do this in the `RRTConnectPlanner` wrapper class.